src/lib/turtle.js

```diff
@@ -1,6 +1,9 @@
 var $builtinmodule = function (name) {
 "use strict";
 
+var DEFAULT_MOUSE_EVENT_LISTENER_OFF_NOTIF_NAME = "SkTurtleMouseEventListenerOff";
+var DEFAULT_TIMER_EVENT_LISTENER_OFF_NOTIF_NAME = "SkTurtleTimerEventListenerOff";
```

Comment:
I'm not sure we necessarily need these vars, if their only purpose is to be assigned on lines 39-40.  I think we could just put the actual strings there.